src/shogun/machine/gp/FITCInferenceMethod.cpp

```diff
@@ -1,17 +1,34 @@
 /*
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 3 of the License, or
- * (at your option) any later version.
- *
+ * Copyright (c) The Shogun Machine Learning Toolbox
+ * Written (W) 2015 Wu Lin
  * Written (W) 2013 Roman Votyakov
- * Copyright (C) 2012 Jacob Walker
```

Comment:
It seems that Roman used Jacob's framework.
In fact, I changed most of the code in `FITCInferenceMethod`